ops/ccs-ops-misc/load_test/common/data.py

```diff
@@ -70,15 +66,40 @@ def load_mbis():
         num_workers = os.environ['LOCUST_NUM_WORKERS']
         print(f"Worker {worker_number} starting...")
         configFile = config.load()
-        full_mbi_list = mbi.loadData()
+        full_mbi_list = db.get_hashed_mbis(configFile['dbUri'])
+        data_per_user = len(full_mbi_list) // int(num_workers)
+        start_index = int(worker_number) * data_per_user
+        end_index = start_index + data_per_user - 1
+        print(f"Worker {worker_number} using data from indexes {start_index} to {end_index}")
+        return full_mbi_list[start_index:end_index]
+    else:
+        configFile = config.load()
+        return db.get_hashed_mbis(configFile['dbUri'])
+
+'''
+Gets the hashed partially adjudicated mbi data and either returns all the data in a list if
+not a distributed test, or takes a percentage of the data to distribute to the current worker
+thread. The percentage of the data in distributed mode depends on the total number of workers
+and the index of the data is dependant on which worker index calls this method.
+'''
+def load_pa_mbis():
```

Comment:
🍰  Seems like since this method seems 99% the same as the one above, you could just pass some data into the method to determine which query gets used (adjudicated vs non-adjudicated data)

Reply:
There is for sure a lot of duplication in this module.  We tried to walk the line of cleaning up a couple of things, but not too much to overwhelm and/or distract the reviewers from the objective at hand.  However, if you feel it's appropriate, I can DRY this module up a bit to reduce duplication.

Reply:
Not a huge deal, just seemed like a chance to consolidate. If you feel like it's out of scope it can be cleaned up later.